airbrake/handler.py

```diff
@@ -25,7 +25,7 @@ class AirbrakeHandler(logging.Handler):
     # pylint: disable=too-many-arguments
 
     def __init__(self, airbrake=None, level=logging.ERROR, project_id=None,
-                 api_key=None, environment=None, base_url=None, hostname=None):
+                 api_key=None, **config):
```

Comment:
So I think this warrants a lot more discussion, and I've created this issue for that: https://github.com/airbrake/airbrake-python/issues/57. For now, I've moved configuration values into arguments with defaults, and then anything that's just for context gets set in the variable args.

I'd like to merge this for the time being, until a better idea is pretty clear.